src/components/Tooltip.js

```diff
@@ -0,0 +1,26 @@
+import ReactTooltip from 'react-tooltip';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons'
+import styled from 'styled-components';
+
+const QuestionMark = styled.div`
+  display: inline-block;
+  color: darkgrey;
+  position: absolute;
+  right: -25px;
+  top: 5px;
```

Comment:
@brampauwelyn if you know a better way to display this to the right of the field, feel free to change this

Reply:
Looks good to me.
Normally I would center it using

```suggestion
  top: 50%;
  transform: translateY(-50%);
```
But this doesn't work well since the form has some margin bottom